tests/test_dvae_winci2020.py

```diff
@@ -84,6 +86,28 @@ def forward(self, x: torch.Tensor) -> torch.Tensor:
 
         self.dvaes = {i: DVAE(self.encoders[i], self.decoders[i]) for i in latent_dims_list}
 
+        # Now we also create a DVAE with a trainable Encoder
+        def deterministic_latent_to_discrete(logits: torch.Tensor, n_samples: int) -> torch.Tensor:
+            # straight-through estimator that maps positive logits to 1 and negative logits to -1
+            hard = torch.sign(logits)
+            soft = logits
+            result = hard - soft.detach() + soft
+            # Now we need to repeat the result n_samples times along a new dimension
+            return repeat(result, "b ... -> b n ...", n=n_samples)
```

Comment:
Do we absolutely need `repeat` here? Seems a bit cumbersome to add einops as a test dependency just for this test. 🤔